api/node/routes.go

```diff
@@ -168,6 +172,8 @@ func StatusMetrics(c *gin.Context) {
 	}
 
 	details := facade.StatusMetrics().StatusMetricsMapWithoutP2P()
+	details[AccStateCheckpointsKey] = facade.GetNumCheckpointsFromAccountState()
```

Comment:
we can think of a different approach because all the metrics excluding these two are fetched from StatusMetrics. Can remain like this for now

Reply:
Ok.